lib/BloqadeSchema/test/execute.jl

```diff
@@ -6,16 +6,50 @@ using JSON
 
 
 @testset "to_schema" begin
-    T = 4
-    atoms = [(i,0) for i in 1:10]
+    T = 2
+    atoms = [(π*i,0) for i in 1:10]
+
+    # values = [1.0,nothing,Waveform(t->sin(2π*t/T)^2,T/2)]
+    values = [1.0,constant(;duration=T,value=1.0),Waveform(t->sin(2π*t/T)^2,T)]
+    params = BloqadeSchema.SchemaConversionParams()
+
+    check_atom_res = x->all(BloqadeSchema.check_resolution.(params.atom_position_resolution,x))
+    check_clock_res = x->all(BloqadeSchema.check_resolution.(params.rabi_time_resolution,x))
+    check_Δ_res = x->all(BloqadeSchema.check_resolution.(params.rabi_detuning_resolution,x))
+    check_Δi_res = x->all(BloqadeSchema.check_resolution.(params.rabi_detuning_local_resolution,x))
+    check_ϕ_res = x->all(BloqadeSchema.check_resolution.(params.rabi_frequency_phase_resolution,x))
+    check_Ω_res = x->all(BloqadeSchema.check_resolution.(params.rabi_frequency_amplitude_resolution,x))
 
-    values = [1.0,nothing,Waveform(t->sin(2π*t/T)^2,T/2)]
     for Ω in values
         for ϕ in values
             for Δ in values
```

Comment:
```suggestion
    for Ω in values, ϕ in values, Δ in values
```

might be better to save a few column by using this more compact form